src/hub.js

```diff
@@ -29,6 +70,31 @@ export function hub(chrome) {
 	 */
 	const appPongs = new Map();
 
+	const appSpawns = [];
+
+	const add = (method, handler) => {
+		methodHandlers.set(method, handler);
+	};
+	const call = (method, argsArr) => {
+		if (methodHandlers.has(method)) {
+			return methodHandlers.get(method).apply({}, argsArr);
```

Comment:
If you pass `null`, there's a chance that if we're not in strict mode, `this` would point to `window`, which I wanted to avoid.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply#Parameters